server/methods.js

```diff
@@ -1,15 +1,52 @@
 Meteor.startup(function() {
   Meteor.methods({
     addComment: function(initiativeId, input) {
+      check(initiativeId, String);
```

Comment:
@mariocatch @iamlasse thanks! yeah `check()` is actually from MDG. It is to check input against what we know that param should be. This can help us catch bugs and stop bogus input from being used. I made a ticket about implementing it in all methods the other day (#53). The MDG docs for check are found here http://docs.meteor.com/#/full/check. The check function you are talking about is `Throttle.check` which isn't used thus far. We always use `Throttle.checkThenSet`